samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -179,6 +179,19 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
 
   const shouldHideScreenShare = isMobileSession || isIOS();
 
+  /* @conditional-compile-remove(file-sharing-acs) */
+  const attachmentOptions = useMemo(() => {
+    // Returning undefined for none group call locators
+    // This includes teams meeting link and teams meeting id locators
+    // Because BYOS file sharing in interop chat is not supported currently
+    if (locator && !isGroupCallLocator(locator)) {
+      return undefined;
+    }
+    return {
+      uploadOptions: attachmentUploadOptions
+    };
+  }, [locator]);
```

Comment:
`attachmentOptions` probably should include `attachmentUploadOptions` in dependencies array